script.js

```diff
@@ -0,0 +1,82 @@
+
+var x = "x";
+var o = "o";
```

Comment:
While this isn't wrong, you probably don't need to make vars for these, since they'll never change. Just use a string literal `"o"` and `"x"` where you need them (e.g. line 25 / 28 below)